process/block/metablock.go

```diff
@@ -405,6 +410,10 @@ func (mp *metaProcessor) CommitBlock(
 		return err
 	}
 
+	log.Info(fmt.Sprintf("starting commit block with round %d and nonce %d\n",
```

Comment:
It helps on sync when we don't know if the sync of one header has not been done because of an error from processBlock or from commitBlock, as there are many common errors which could be returned from these both methods. It will be only one log per round in a normal case.  But for now I will put them on Debug level.